overlord/assertstate/validation_set_tracking_test.go

```diff
@@ -133,12 +133,12 @@ func (s *validationSetTrackingSuite) TestUpdate(c *C) {
 	c.Check(gotSecond, Equals, true)
 }
 
-func (s *validationSetTrackingSuite) TestDelete(c *C) {
+func (s *validationSetTrackingSuite) TestForget(c *C) {
```

Comment:
maybe this test should have a comment saying there's a more extensive test in assertstate_test.go as well